.s2i/bin/run

```diff
@@ -7,32 +7,5 @@
 #	https://github.com/openshift/source-to-image/blob/master/docs/builder_image.md
 #
 
-set -e
-id
-whoami
 
-echo "Running app..."
-
-echo "PWD = "$PWD
-echo "ls -l /opt/app-root/src/"
-ls -l /opt/app-root/src/
-#echo "MODEL_NAME = "$MODEL_NAME
-#echo "MODEL_PATH = "$MODEL_PATH
-#echo "model_base_path = /opt/app-root/src/"
-
-echo "ls -l /opt/app-root/"
-ls -l /opt/app-root/
-
-#export CUDA_HOME="/usr/local/cuda"
-#export CUDA_PATH="${CUDA_HOME}"
-#export PATH="${CUDA_HOME}/bin${PATH:+:${PATH}}"
-#export LD_LIBRARY_PATH="${CUDA_HOME}/lib64${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}";
-#echo -e “$CUDA_HOME \\n $CUDA_PATH \\n $LD_LIBRARY_PATH”
-
-#echo " -----> Run tensorflow server."
-
-#binary gets copied to  /opt/app-root/src/tf/tensorflow_model_server
-#/opt/app-root/src/tf/tensorflow_model_server --port=6006 --model_name=$MODEL_NAME --model_base_path=/opt/app-root/src/$MODEL_PATH
-
-echo "---> Running Inference:"
```

Comment:
I'm fine with removing all the commented out code here, but would argue that the echo statements were quite useful when getting this sample to work and debugging things and if I was to use this as a sample would immediately be having to add that sort of thing back in.
I'd argue to re-add Lines, 10-12, 14-18 and 23-24 back in. 
I'm not sure why you removed running inference from the run script -- I figured the point of this sample was to show someone a basic app they can build from and by omitting this it makes it a little harder for a newbie to properly launch an app. Much like the comment you added in the readme they can override this defailt